### DISCUSSION TITLE
Hosting Documentation on a Dedicated Website

### DISCUSSION THREAD

**Discussion by SethBurkart123** [open-webui/open-webui#824] [Ideas]
Hello Everybody,

I've noticed that the documentation is not currently hosted on a dedicated website, which could limit its accessibility and ease of use. Hosting the documentation on a website can significantly enhance user experience by providing better navigation, search capabilities, and a more engaging interface.

But it can be hard to create a documentation site. But I have an idea for that. If interested I'd be happy to bring in Tailwind UI's protocol template (https://protocol.tailwindui.com/) which uses nextjs and mdx. I already have access to it and it could easily be set up for the project. It would just provide a much nicer experience to new users coming in and trying to figure out how to get started - As well as documentation for more complicated features (many still in the roadmap) such as extending the backend or writing custom functions for LLMs.

**Comment by tjbck** [comment]
We actually have an on-going effort which we just added a couple of days ago: https://github.com/open-webui/docs I'll be adding more contents to the docs overtime, feel free to make a PR in the docs repo!

**Reply by SethBurkart123** [reply]
Oh! Great, well that's all good. I'll have a look and see whether I can help out with anything.
